crates/uv-client/src/registry_client.rs

```diff
@@ -459,6 +459,10 @@ impl RegistryClient {
             .client
             .uncached()
             .head(url.clone())
+            .header(
+                "accept-encoding",
+                http::HeaderValue::from_static("identity"),
```

Comment:
Iiinteresting, so when the server returned a gzipped response here, we effectively lost the headers that are required for the range request?

Reply:
I wonder if this is related: https://github.com/astral-sh/uv/issues/1754.

Reply:
Eh, that issue suggests that `Content-Type` is preserved, nevermind.

Reply:
Yep they get stripped because they're not accurate after decompression https://github.com/seanmonstar/reqwest/issues/294